utility/src/test/java/org/oppia/android/util/namevalidator/NameValidatorTest.kt

```diff
@@ -0,0 +1,27 @@
+package org.oppia.android.util.namevalidator
+
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
```

Comment:
Please use Truth assertions, instead (see uses of ``assertThat`` elsewhere in the codebase).